### ISSUE TITLE
Using the topic benefit of MQTT for Measurement Mappings

### ISSUE THREAD

**Issue by mbay-ODW** [thin-edge/thin-edge.io#509]
**Is your feature request related to a problem? Please describe.**
Currently the tedge/measurements awaits an json based payload. However this does not make benefit of the topic structure provided by MQTT. Many gateways, sensors, implementations, programs do send their data depending on their meaning e.g. sensor1/temperature/internal with a json or non json based payload. 

**Describe the solution you'd like**
It would be great to provide next to tedge/measurements also the possibility to have tedge/measurements/x/y that creates the measurement mapping depending on the used topic structure for measurements and series. A good approach for e.g.

tedge/measurement/sensor1/temperature/internal and the payload: {value: 12, unit: "°"} could be:

{
"sensor1/temperature":
               {"internal": 
               {"value": 12, "unit": "°"}}
}

or 

{
"sensor1":
               {"temperature/internal": 
               {"value": 12, "unit": "°"}}
}

So basically using the last or everything except the first as series and everything else as measurement. However the example is just an idea.
**Describe alternatives you've considered**
The current concept relays that the device already builds up a logic within the payload such that logic or implementation is needed. However many devices allow the configuration of blank payloads that are just send on various topics where currently implementation effort as additional module is required.

**Comment by jvanderl** [comment]
Interesting feature if you ask me. Currently the first level in the topic after "tedge/measurement/" is reserved for the child device id, so we would have to introduce something like "tedge/child/<childID>/measurement", and then do the same as described above for the sensor layering by topic.

**Comment by mbay-SAG** [comment]
Any feedback on that? Really would like to discuss the idea behind using the topic benefits.

**Comment by didier-wenzek** [comment]
There is a work ~~in progress~~ stalled about this. see https://github.com/thin-edge/thin-edge.io-specs/pull/19

**Comment by mbay-SAG** [comment]
Why was this stalled here without any feedback?

**Comment by didier-wenzek** [comment]
> Why was this stalled here without any feedback?

There is a very simple reason for the lack of feedback: I'm overwhelmed.

I agree that we have to improve the data model for measurements - this request is one of the points to address, there is also the issue related to measurement units #982. Unfortunately, we also have to manage priorities.
